project_case_default/README.rst

```diff
@@ -0,0 +1,75 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+====================
+Project Case Default
```

Comment:
I think that this name can create confusion between project stage and task stages. The default stage is not for project, is for task. 
What do you think about *project_task_default_stage*?